### ISSUE TITLE
unknown[92.246.76.92]: SASL LOGIN authentication failed: UGFzc3dvcmQ6

### ISSUE THREAD

**Issue by arekstuka** [whyscream/postfix-grok-patterns#137]
HI  ,

Dear all, 
During the testing GROK for postfix i found one thing what can be helpfull for analize postix logs. 
Right now If somebody try to access the mailserver with wrong user or password. inside the field "postfix_message" we can find below info:

postfix_message |   | unknown[92.246.76.92]: SASL LOGIN authentication failed: UGFzc3dvcmQ6

Will be nice if we ca also parse attacker IP address. and message "SASL LOGIN authentication failed: UGFzc3dvcmQ6"
IF somebody from the users already solve this issue, please let me know. 

thx

Arek

**Comment by whyscream** [comment]
A full line looks like: 
```
Mar 10 12:08:39 hostname postfix-in/submission/smtpd[8871]: warning: ec2-3-84-57-208.compute-1.amazonaws.com[3.84.57.208]: SASL LOGIN authentication failed: UGFzc3dvcmQ6
```

We currently already try to extract a queue id and various key-value data from a warning message. I guess it would be possible to extract the client info too, similar to all the other line that contain client details.

**Comment by whyscream** [comment]
A full line looks like: 
```
Mar 10 12:08:39 hostname postfix-in/submission/smtpd[8871]: warning: ec2-3-84-57-208.compute-1.amazonaws.com[3.84.57.208]: SASL LOGIN authentication failed: UGFzc3dvcmQ6
```

We currently already try to extract a queue id and various key-value data from a warning message. I guess it would be possible to extract the client info too, similar to all the other line that contain client details.

**Comment by arekstuka** [comment]
> A full line looks like:
> 
> ```
> Mar 10 12:08:39 hostname postfix-in/submission/smtpd[8871]: warning: ec2-3-84-57-208.compute-1.amazonaws.com[3.84.57.208]: SASL LOGIN authentication failed: UGFzc3dvcmQ6
> ```
> We currently already try to extract a queue id and various key-value data from a warning message. I guess it would be possible to extract the client info too, similar to all the other line that contain client details.

Did You update the master file in github with Your patterns? If not maybe it is a good idea, or maybe you can just share it? Basicly it is nice to see the potentialy attacker ip in separate field.

**Comment by whyscream** [comment]
There are no updated files yet. What I described is already in master. Extracting the client details is something that can be added. I'll give it a try when I have some time later this week

**Comment by arekstuka** [comment]
OK.Thanks.
If You need some test before, i can help. Let me know...

**Comment by whyscream** [comment]
I think I already solved it using #138

**Comment by arekstuka** [comment]
I will check it tomorrow and for sure i will let You know.
